build.gradle

```diff
@@ -66,6 +68,11 @@ dependencyUpdates.resolutionStrategy = {
 
 dependencyCheck {
     suppressionFile = 'dependency-check-suppressions.xml'
+    analyzers {
+        ossIndex {
+            enabled = false
+        }
+    }
```

Comment:
Required to match configuration of the Jenkins pipeline, otherwise we'll see a discrepancy in results.

Reply:
Removed as the more recent plugin version,[ 0.12.28](https://github.com/hmcts/rse-gradle-java-plugin/releases/tag/0.12.28), matches the pipeline build. Also, an error may be returned if this code is present alongside the changes from version 0.12.28.